template.yaml

```diff
@@ -857,6 +887,239 @@ Resources:
             Unit: Count
   #endregion
 
+
+  AuthorisationFunction:
+    Type: AWS::Serverless::Function
+    # checkov:skip=CKV_AWS_116: DLQ is not appropriate for a Lambda invoked by an API
+    # checkov:skip=CKV_AWS_117: Lambdas will migrate to our own VPC in future work
+    IgnoreGlobals: [ VpcConfig ]
+    Properties:
+      FunctionName: !Sub ${Environment}-AuthorisationFunction
+      AutoPublishAlias: latest
+      CodeUri: ./oidc-api
+      Handler: uk.gov.di.authentication.oidc.lambda.AuthorisationHandler::handleRequest
+      ReservedConcurrentExecutions: 1
+      LoggingConfig:
+        LogGroup: !Ref AuthorisationFunctionLogGroup
+      VpcConfig:
+        SecurityGroupIds:
+          - !Ref LambdaSecurityGroup
+          - !Ref HttpsEgressSecurityGroup
```

Comment:
Do you mind making a quick change in this PR and adding this to doc app as well?